jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/components/drawer/InfoView.kt

```diff
@@ -126,6 +127,7 @@ class InfoView : View() {
                     textarea {
                         addClass("app-drawer__report-message")
                         textProperty().bindBidirectional(viewModel.errorDescription)
+                        overrideDefaultKeyEventHandler()
```

Comment:
are we missing the action passed in here, like what of the button below?
` viewModel.submitErrorReport() `